docs/docs/0-testkube-cloud/10-architecture.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 10
+sidebar_label: Testkube architecture
+---
+
+## Multiple Testkube Agents
+
+Main Testkube Cloud feature is to have insights into multiple Testkube Cloud Agents. 
+You can look at your Kubernetes clusters from single dashboard. 
+
+
+![multiple clusters](https://user-images.githubusercontent.com/30776/208391158-a42d1f56-950f-48c3-bcfb-2768054b4704.jpeg)
+
+
+## Storing results
+
+In Testkube standalone all results are stored in the users cluster, you need to be aware of MinIO and MongoDB. 
+Testkube Cloud will make it easy for you, all data are stored in Testkube Cloud infrastructure.
+
+
+## Testkube networking
+
+To simplify networking connections Testkube Agent is able to create tunnel to Testkube Cloud clusters. The main 
```

Comment:
not sure, about `tunnel` term there, may confuse users

Reply:
rewritten it a little